docs/references/api/deviceshifu-configmap.md

```diff
@@ -65,6 +65,11 @@ instructions: |
         protocolPropertyList:
           MQTTTopic: "/test/test2"
       ...  # You can continue to configure commands and corresponding topics according to your own needs, just continue to add according to this format
+mutexInstructions: | 
+    # Configure mutexInstructions as required. The format is, mutexInstruction:"description"
```

Comment:
shall we illustration the format as key, value? and tell them what does this config is used for, and what the keys stands for, what value they should put? is there any validation of the key and value?